lib/Connection/ErrorLog.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Domnikl\Statsd\Connection;
+
+use Domnikl\Statsd\Connection as Connection;
```

Comment:
On a separate note: there's a class name that exists in conflict with this namespace?